src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax;
+
+public class RightTriangle extends Figure {
+    private double firstLeg, secondLeg, thirdLeg;
+    private String color;
+
+    public RightTriangle(double firstLeg, String color) {
+        this.firstLeg = firstLeg;
+        this.secondLeg = firstLeg;
+        this.thirdLeg = firstLeg;
+        this.color = color;
+    }
+
+    public RightTriangle() {
+    }
+
+    @Override
+    public double findArea() {
+        return (firstLeg * firstLeg * Math.pow(3, 2)) / 4;
+    }
+
+    @Override
+    public String printAllInformation() {
+        return null;
+    }
```

Comment:
The `RightTriangle` class should only have two legs as instance variables since the third leg (hypotenuse) can be calculated from the first two. Also, the `color` should not be a part of this class as it's already handled by the `Figure` class which this one extends.